frontend/src/store/summaryDeclaration.js

```diff
@@ -433,8 +433,8 @@ export const useSummaryDeclarationStore = defineStore('summaryDeclaration', {
     async loadChangeRequestSummaryForChangeNotiForm(payload) {
       const reportChangesStore = useReportChangesStore();
       try {
-        let summaryModel = this.summaryModel;
-        let changeNotiChangeAction = payload.changeActions?.find(
+        const summaryModel = this.summaryModel;
```

Comment:
I feel like this assignment isn't necessary but we can leave as is for now.

Reply:
sounds good. I didn't update any logic here - I was just cleaning up some of the logs and let/consts